lib/config/v2/gl_eink.ex

```diff
@@ -23,7 +23,11 @@ defmodule ScreensConfig.V2.GlEink do
             alerts: nil,
             line_map: nil,
             evergreen_content: [],
-            audio: Audio.from_json(:default),
+            audio: %Audio{
+              start_time: ~T[00:00:00],
+              stop_time: ~T[23:59:59],
+              days_active: [1, 2, 3, 4, 5, 6, 7]
+            },
```

Comment:
I think you meant to add this to `Busway`?

Reply:
🤦

Reply:
I suppose we could have a `Audio.always/0` that returns this value, and use that as the default for all the push-button kinds of screens.

Reply:
Yeah not a bad idea.

Reply:
Went ahead and changed GL E-Ink and Pre-Fare to use that too. One less step for Ana when she makes more of those screens.

Reply:
As long as that won't mess with Permanent Config setup of GL E-ink screens.

Reply:
That tool creates the struct with these values so the default is never used.